astor/code_gen.py

```diff
@@ -226,6 +226,32 @@ def body_or_else(self, node):
         self.body(node.body)
         self.else_body(node.orelse)
 
+    def docstring(self, node, indent=1):
+        """
+        Write the function/module/class docstring attribute
+
+        This method has an effect only for python >= 3.7 . For python earlier
+        versions the docstring was represented as the first expression in the
+        function/module/class body.
+
+        Args:
+            node (ast.AST): The node whose docstring is printed.
+            indent (int): How much to indent the docstring (usually is 0 for
+                ast.Module, and 1 for ast.FunctionDef, ast.AsyncFunctionDef and
+                ast.ClassDef).
+        """
+        if not sys.version_info >= (3, 7):
```

Comment:
I'm pretty sure that we are going to keep the `docstring` attribute in the future Pythons so can't we just write `if sys.version_info < (3, 7):` here?

Reply:
Ok, I simplified it to
```python
        if sys.version_info < (3, 7):
            return
```